SocialFilter/sections/specific.txt

```diff
@@ -46,6 +46,7 @@ originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,analogictips.co
 ! ##.sns_area
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
+mens.tasclap.jp##.p-article-main-bottom-share
```

Comment:
Better `mens.tasclap.jp##.p-article-main-bottom-share::before`.

Reply:
ok,thank you for advice